mappings/net/minecraft/client/render/entity/WitherSkullEntityRenderer.mapping

```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/class_966 net/minecraft/client/render/entity/WitherSkullEntityRenderer
-	FIELD field_4815 SKIN Lnet/minecraft/class_2960;
+	FIELD field_4815 TEXTURE Lnet/minecraft/class_2960;
 	FIELD field_4816 model Lnet/minecraft/class_607;
-	FIELD field_4817 INVINCIBLE_SKIN Lnet/minecraft/class_2960;
+	FIELD field_4817 INVINCIBLE_TEXTURE Lnet/minecraft/class_2960;
```

Comment:
Similar question to invulnerable Vs invincible

Reply:
```suggestion
	FIELD field_4817 INVULNERABLE_TEXTURE Lnet/minecraft/class_2960;
```